openedx/openedx-platform
# update login lockout params

### [PROD-361](https://openedx.atlassian.net/browse/PROD-361)

### Description
This PR updates the following pararms:
- MAX_FAILED_LOGIN_ATTEMPTS_ALLOWED
- MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS

### Sandbox
 - https://lockout.sandbox.edx.org/

### Testing Instructions
 1. Visit the sandbox
 2. Navigate to Login Page
 3. Use either verified@example.com or audit@example as the email.
 4. Try to login with an incorrect password 6 times.
 5. After the threshold, the banner would say the account is temporarily locked.
 6. Try logging in with the correct password. The attempt will be unsuccessful.
 7. After 30 minutes, try to log in and confirm if the login is successful.

### Reviewers
 - [x] @noraiz-anwar 

### Post Review
